src/status_im/network/net_info.cljs

```diff
@@ -11,10 +12,13 @@
 
 (defn- wrap-net-info [callback]
   (fn [info-js]
-    (let [info (js->clj info-js :keywordize-keys true)]
-      (.then (.isConnectionExpensive react-components/net-info)
-             (fn [expensive?]
-               (callback {:type (:type info) :expensive? expensive?}))))))
+    (let [info      (js->clj info-js :keywordize-keys true)
+          callback' #(callback {:type (:type info) :expensive? %})]
```

Comment:
Can we find a better name? Or extract it as a dedicated fn?

Reply:
Could you suggest a better name?

Reply:
Not really. Anything as long as there is no potential confusion with another argument.

Reply:
we could name it something like `wrapped-callback` / `prepared-callback`, a bit verbose but explicit, or name the parameter `raw-callback`.

Just to understand, in general are we moving away from the haskell convention of naming a modified version of the var by appending `'` ( I see it's used in other parts of the codebase) ?

I might start capturing this in a doc as it might be useful for newbies like me :)

Reply:
Haskell ? :fearful: 
How about using clojure conventions? :smile: 

Please @cammellos start such doc in the wiki!

Reply:
It'd be interesting to know if there's a convention for such thing in clojure, I have always used `raw-` and `prepared-`, but worth having a look around.

I will start a doc so we can be all on the same page.